### PR TITLE
Add ADO build config files.

### PR DESCRIPTION
**Pull request by praries880**
This will enable us to use ADO for CI and release.

### CODE REVIEW

File: .azure-pipelines/mgmt.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+pr:
```
Comment (weshaggard) [inline .azure-pipelines/mgmt.yml]: I would rename the mgmt.yml to represent what you are building. Perhaps fluent?

File: .azure-pipelines/mgmt.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+variables:
+  DotNetCoreVersion: '2.1.503'
+
+jobs:
+  - job: Windows_Build
+    pool:
+      vmImage: 'VS2017-Win2016'
+    variables:
+      IgnoreBuildTokens: ''
```
Comment (weshaggard) [inline .azure-pipelines/mgmt.yml]: If you don't use this you can just remove it here and in the msbuildARguments below.

**Comment by weshaggard** [comment]
Can you send me a link to a test run you have kicked off?

File: .azure-pipelines/fluentNetSdk.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+pr:
+  branches:
+    include:
+    - master
+    - search-preview
```
Comment (weshaggard) [inline .azure-pipelines/fluentNetSdk.yml]: These branches don't look interesting to your repo. I would remove all but master.

Comment (praries880) [reply .azure-pipelines/fluentNetSdk.yml]: done

**Pull request merged**